libraries/core-react/src/Chip/Chip.tsx

```diff
@@ -124,27 +124,38 @@ const StyledChips = styled.div.attrs(({ clickable, deletable }) => ({
       padding-right: 4px;
     `}
 
-  ${({ onlyChild }) =>
-    onlyChild &&
+  ${({ children }) =>
+    (typeof children === 'string') &&
     css`
       padding-left: 8px;
     `}
-
 `
-
-export const Chip = forwardRef(function EdsChips(
-  { children, onDelete, disabled, onClick, variant, ...rest },
+type Props = {
+  disabled?: boolean,
```

Comment:
Would you please add a `/** Comment describing what this prop is */` above each prop? When we render this in a PropTable-component in the storybook and in eds.equinor.com it adds the comment in the documentation

Reply:
Have added comments from the old PropTypes for the ones that we still specify. Some of them have been removed as they are standard Div properties.

Reply:
👍